bin/test-ci

```diff
@@ -7,4 +7,6 @@ set -ev
 
 $(npm bin)/jest --maxWorkers=2 --verbose --coverage --config ./configs/test/jest.$OX_PROJECT.json
 
+echo -en 'travis_fold:start:build\\r'
```

Comment:
ooh, yeah! I love this little "hidden" feature in Travis 😄

Reply:
Yeah, it's nice for stuff like this where it generates a huge amount of text